src/index.js

```diff
@@ -27,6 +28,63 @@ function generateCopyTarget(src, dest, rename) {
   }
 }
 
+function generateCopyTargets(src, dest, rename) {
+  return Array.isArray(dest)
+    ? dest.map(destination => generateCopyTarget(src, destination, rename))
+    : [generateCopyTarget(src, dest, rename)]
+}
+
+async function copyFiles(copyTargets, verbose, copyOptions) {
+  if (Array.isArray(copyTargets) && copyTargets.length) {
+    if (verbose) {
+      console.log(green('copied:'))
+    }
+
+    for (const { src, dest } of copyTargets) {
+      await fs.copy(src, dest, copyOptions)
+
+      if (verbose) {
+        console.log(green(`  ${bold(src)} → ${bold(dest)}`))
+      }
+    }
+  } else if (verbose) {
+    console.log(yellow('no items to copy'))
+  }
+}
+
+function watchFiles(targets, verbose, copyOptions) {
+  return targets.map(({ src, dest, rename }) => {
+    async function onChange(matchedPath) {
+      const copyTargets = generateCopyTargets(matchedPath, dest, rename)
+      await copyFiles(copyTargets, verbose, copyOptions)
+    }
+
+    return chokidar.watch(src, { ignoreInitial: true })
```

Comment:
Test added - see 'Watches and copies multiple targets from same file'

Reply:
Yeah, I understand why multiple watchers. So, basically 5 targets - 5 watchers. Not ideal, but solves problems with understanding what target was changed. I'll think if this can somehow be improved and stick with multiple watchers if nothing simple to create comes to mind.

Reply:
so basically, if a target has a glob with for example 100 files and one of them is changed, we copy 100 files again?

Reply:
No.

So basically, we will tell chokidar to watch the glob matching 100 files, and when one of them changes, chokidar will call our callback with the path for that single file. Then we will generate copy targets for that file (it may need to be copied to more than one destination), and copy it to the required destinations.

Reply:
Nice! 👍